### PR TITLE
tsort: move from getopts to clap

### PR DESCRIPTION
**Pull request by ycd**
closes: https://github.com/uutils/coreutils/issues/1862

### CODE REVIEW

File: tests/by-util/test_tsort.rs [hunk 1]
```diff
@@ -15,3 +15,36 @@ fn test_sort_self_loop() {
...
+fn test_no_such_file() {
+    let result = new_ucmd!().pipe_in("invalid_file_txt").run();
+
+    assert_eq!(true, result.stdout.contains("No such file or directory"));
+}
+
+#[test]
+fn test_no_such_file() {
+    let version_short = new_ucmd!().arg("-V").run();
+    let version_long = new_ucmd!().arg("--versioon").run();
```
Comment (ycd) [inline tests/by-util/test_tsort.rs:29]: ups typo.

File: src/uu/tsort/src/tsort.rs [hunk 2]
```diff
@@ -9,49 +9,36 @@
 #[macro_use]
 extern crate uucore;
 
+use clap::{App, Arg};
 use std::collections::{HashMap, HashSet};
 use std::fs::File;
 use std::io::{stdin, BufRead, BufReader, Read};
 use std::path::Path;
 
 static NAME: &str = "tsort";
```
Comment (sylvestre) [inline src/uu/tsort/src/tsort.rs:18]: please remove NAME :)

**Comment by sylvestre** [comment]
Fails on Windows with:
```

---- test_tsort::test_multiple_arguments stdout ----
current_directory_resolved: 
run: D:\a\coreutils\coreutils\target\x86_64-pc-windows-msvc\debug\coreutils.exe tsort call_graph.txt invalid_file.txt
thread 'test_tsort::test_multiple_arguments' panicked at 'assertion failed: `(left == right)`
  left: `true`,
 right: `false`', D:\a\coreutils\coreutils\tests\by-util\test_tsort.rs:49:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

---- test_tsort::test_no_such_file stdout ----
current_directory_resolved: 
run: D:\a\coreutils\coreutils\target\x86_64-pc-windows-msvc\debug\coreutils.exe tsort
thread 'test_tsort::test_no_such_file' panicked at 'assertion failed: `(left == right)`
  left: `true`,
 right: `false`', D:\a\coreutils\coreutils\tests\by-util\test_tsort.rs:23:5
```

**Comment by ycd** [comment]
@sylvestre yeah, fixed it, i was checking stdout for errors :(

**Comment by sylvestre** [comment]
Kudos for the tests :)

**Pull request merged**